rules/codelingo/effective-go/avoid-annotations-in-comments/codelingo.yaml

```diff
@@ -27,14 +27,14 @@ tenets:
           The package comment for the fmt package uses this to good effect.
           
           ## Bad code
-          ```
+          ```golang
           /**
           * This is a bad_comment_example
```

Comment:
The underlines mentioned are used to indicate italics or emphasis. This only works when there are spaces on either side. The other example in the query is someone trying to use html tags.

The comment should read like a normal comment where someone might think to use emphasis. Just making something dumb up here:
```
// UnsmartenQuotes replaces <code>“smart quotes”</code> with _standard_ <code>"quotes"</code>.
```

Equivalent good code:
```
// UnsmartenQuotes replaces “smart quotes” with standard "quotes".
```